File: GoLang/gradecalc.go
//write the template for a go console app
package main

import (
	"fmt"
)

func main() {
	var BD1, CA, EN3, P3, PI, SO1, SDA, TGC int = 7, 10, 9, 8, 9, 8, 9, 10
	var credit3, medie3 float64
	credit3 = 5*float64(TGC) + 5*float64(SO1) + 5*float64(BD1) + 5*float64(P3) + 4*float64(SDA) + 2*float64(PI) + 2*float64(EN3) + 2*float64(CA)
	medie3 = credit3 / 30
	fmt.Printf("Medie ponderata: %.3f", medie3)
}

File: Dart/gradecalc.dart
void main() {
int bd1 = 7, ca = 10, en3 = 9, p3 = 8, pi = 9, so1 = 8, sda = 9, tgc = 10;


  double credit3, medie3;
    credit3 = 5.0*tgc + 5.0*so1 + 5.0*bd1 + 5.0*p3 + 4.0*sda + 2.0*pi + 2.0*en3 + 2.0*ca;

  medie3 = credit3 / 30;
  print("Medie ponderata: ${medie3.toStringAsFixed(3)}");
}